config/webpack.config.babel.js

```diff
@@ -0,0 +1,85 @@
+/* @noflow */
+/* eslint-disable import/no-extraneous-dependencies */
+
+import webpack from 'webpack';
+import HtmlWebpackPlugin from 'html-webpack-plugin';
+import CopyWebpackPlugin from 'copy-webpack-plugin';
+import path from 'path';
+import {
+	commonConfig,
+	uglifyPlugin
+} from './common.webpack.config.babel';
+
+const ENV = process.env.NODE_ENV || 'development';
+
+
+module.exports = [
+	// CMP
+	{
+		entry: {
+			cmp: './index.js',
+			'cmp.complete': './complete.js'
+		},
+
+		output: {
+			path: path.resolve(__dirname, '../', 'dist'),
+			publicPath: './',
+			filename: '[name].bundle.js'
+		},
+		...commonConfig,
+		plugins: [
+			new webpack.NoEmitOnErrorsPlugin(),
+			new webpack.DefinePlugin({
+				'process.env.NODE_ENV': JSON.stringify(ENV)
+			}),
+			new webpack.ProvidePlugin({
+				Promise: 'promise-polyfill'
```

Comment:
The babel-polyfill tweak should allow you to remove this

Reply:
(see test plan) i removed babel-polyfill because it bloats the library (by 130k), will make a ticket to figure out how separate out the polyfills or optimize this